development.md

```diff
@@ -0,0 +1,8 @@
+# Gazebo Development
+
+The following guides describe how Gazebo is developed, maintained, and
+released.
+
+* [Continuous Integration](ci)
+* [Contributing](contributing)
```

Comment:
It looks like the list is in alphabetical order, but for a contributor most likely they will look at the `contributing` page first.

Reply:
reordered